uint32 tree_num
uint16[] attr_split
float32[] val_split
int32[] r_node_inds
uint32[] out_of_bags
uint16 num_classes
